.cypress/integration/2_notebooks.spec.js

```diff
@@ -49,9 +49,9 @@ describe('Adding sample data and visualization', () => {
     cy.get('.euiModalHeader__title[data-test-subj="confirmModalTitleText"]')
       .contains('Add samples')
       .should('exist');
-    cy.wait(100);
-    cy.get('.euiButton__text').contains('Yes').trigger('mouseover').click();
     cy.wait(100 * 5);
```

Comment:
Minor: question what's difference between 100 * 5 vs 500? also could we avoid hard waiting here instead to have something maybe a timeout?